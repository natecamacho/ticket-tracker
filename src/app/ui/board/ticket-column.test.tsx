import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import TicketColumn from "./ticket-column";

describe("TicketColumn component", () => {
  it("renders initial TicketColumn component unchanged", () => {
    const { container } = render(<TicketColumn />);

    expect(container).toMatchSnapshot();
  });
});
