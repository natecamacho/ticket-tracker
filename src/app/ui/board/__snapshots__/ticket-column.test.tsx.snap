// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TicketColumn component renders initial TicketColumn component unchanged 1`] = `
<div>
  <div
    class="w-80 h-4/5 bg-slate-200 mr-4 mt-12 pt-2 text-center rounded-md"
  />
</div>
`;
